spec/models/user_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'spec_helper'
+require 'rails_helper'
+RSpec.describe User, type: :model do
+  let(:user) do
+    User.new(email: 'test@example.com', guest: false) { |u| u.save!(validate: false)}
+  end
+
+  it "has an email" do
+    expect(user.email).to be_kind_of String
+    expect(user.email).to eq ("test@example.com")
+  end
+
+  it "has mailbox-related methods defined" do
+    expect(user).to respond_to(:mailboxer_email)
+    expect(user).to respond_to(:name)
```

Comment:
I think one more, which test the return value expected `expect(user.mailboxer_email).to eq user.email`